modules/dashboard/widgets/dashboard-notepad.php

```diff
@@ -59,9 +59,6 @@ public function handle_notepad_update() {
 		} else {
 			wp_insert_post( $new_note );
 		}
-
-		wp_safe_redirect( wp_get_referer() );
```

Comment:
Is this change expected to be part of this PR?

Reply:
Yes. When data was sent over GET, that line had the use of hiding the querystring. The user was immediately redirected to the page it created the note from, so that he would not see the GET parameters. However, [wp_get_referer returns false after a POST request](https://codex.wordpress.org/Function_Reference/wp_get_referer), so I had to be taken away, together with the following `exit`.